### ISSUE TITLE
[Feature Request]: Only Show Latest x Videos for Each Channel

### ISSUE THREAD

**Issue by c-ciobanu** [FreeTubeApp/FreeTube#5265]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

I would like to have my `Subscriptions` panel as some sort of overview on all my channels where I can see the latest videos from all the channels I'm subscribed too.
If you are subscribed to many channels you probably have some that upload much more than others cluttering your `Subscriptions` panel just with their videos.
There is the option `Only Show Latest Video for Each Channel` that will kinda do what I'm asking for but only showing 1 video per channel is not enough for me since I can't really understand if that video is the latest or not. In my case I would like to only be able to see the latest 3 or 5 videos from each channel so I can have a nice understanding of what is going on.

### Proposed Solution

Update the existing option `Only Show Latest Video for Each Channel` so that is possible to manually define the maximum amount amount of videos to show per channel, or create a completely new option that allows for that and call it `Only Show Latest x Videos for Each Channel`

### Alternatives Considered

None.

### Issue Labels

improvement to existing feature, new feature, new optional setting

**Comment by c-ciobanu** [comment]
Hi, I started working on this and I was wandering if it's a good idea to get rid of the existing `Only Show Latest Video for Each Channel` settings in favour of this one since it basically does the same thing.

**Comment by kommunarr** [comment]
Yes, the new logic should also be being invoked in the same file & location as the original setting (`subscriptions.js`).

**Comment by Thunderschnozzle** [comment]
Not sure if it's too late to add an additional request to the original one, but I'll do it anyways. Would it be possible to replace "Latest x Videos for Each Channel" with "Latest x Videos for Each Channel from the Last y Period"?
The user would be given a drop-down menu for x that will say: 1,2,3,4,5..., and a drop-down menu for y that will say:1 day, 2 days, 3 days, 1 week, 2 weeks, 1 month...
What do you guys think?
